.circleci/config.yml

```diff
@@ -22,9 +22,12 @@ jobs:
             mkdir -p ~/.ssh && touch ~/.ssh/config && echo $'Host github.com\n\tStrictHostKeyChecking no' > ~/.ssh/config
             git clone git@github.com:AudiusProject/audius-tooling.git
             git clone git@github.com:AudiusProject/mad-dog.git
+            cd /home/circleci/project/
+            git checkout dm-link-dev-server
```

Comment:
@dmanjunath i think we are blocked on this https://github.com/AudiusProject/audius-protocol/pull/787 right

Reply:
so that was the question i had for you @hareeshnagaraj. i know we had talked about maybe adding #787 but i couldn't remember if we said we'd merge it onto master